Index: dz_6.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/dz_6.py b/dz_6.py
--- a/dz_6.py	(revision 41846a0816d6548514e1069df5cb0d9872d53fd6)
+++ b/dz_6.py	(date 1652874092626)
@@ -1,0 +1,5 @@
+from colorama import Fore, Style, Back, Cursor
+
+print(f"{Fore.LIGHTGREEN_EX}\n\tThis text is having a lightgreen color{Fore.BLUE}, but this text is having blue color!\n\n\t{Fore.RED}For change color of text you need to use a function{Fore.YELLOW} \" Fore.COLOR \"{Fore.RED}, where \" COLOR \" — this is color which you want to use!")
+print(f"{Style.RESET_ALL}\n\tBut, this is no all. Library {Fore.CYAN}\" colorama \"{Style.RESET_ALL} supports functions such as: {Fore.LIGHTYELLOW_EX}\" Back \"{Fore.RESET} , {Fore.LIGHTYELLOW_EX}\" Style \"{Fore.RESET} , {Fore.LIGHTYELLOW_EX}\" Cursor \"{Fore.RESET} !")
+print(f"{Fore.GREEN}\n\tFunction {Fore.BLUE}\" Back \"{Fore.GREEN} — circles the text with a certain color on the outside, for example:{Fore.MAGENTA} \" {Back.LIGHTYELLOW_EX}word{Back.RESET} \"{Fore.GREEN}, as you can see the word \" word \" is circled in light yellow color !")
\ No newline at end of file
